jkube-kit/resource/helm/src/main/java/org/eclipse/jkube/kit/resource/helm/HelmService.java

```diff
@@ -323,4 +324,11 @@ private static List<HelmParameter> collectParameters(HelmConfig helmConfig) {
     });
     return parameters;
   }
+
+  private static String resolveHelmClassifier(HelmConfig helmConfig) {
+    if (StringUtils.isEmpty(helmConfig.getTarFileClassifier())) {
```

Comment:
note for any subsequent PR, please use isBlank instead, this way we make sure a classifier such as `  ` is not considered.